Fix T1001631 - Chart - scale breaks are not visible if its range exceeds the tick interval (#2402)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxChart/1 Configuration/argumentAxis/breaks/breaks.md

Declares a scale break collection. Applies only if the axis' [type](/api-reference/10%20UI%20Components/dxChart/1%20Configuration/argumentAxis/type.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#type') is *"continuous"* or *"logarithmic"*.

---

> A scale break is an area across an axis that is displayed instead of a section of an axis range. It allows you to improve the readability of chart sections with large gaps in their ranges.

Reviewer: ```suggestion
A scale break is an area across an axis that is displayed instead of a section of an axis range. Scale breaks improve the readability of chart sections with large gaps in their ranges.
```

Reviewer 2: Sounds good.

> 

Reviewer 2: Is it possible to update line 13 above? I'd like to avoid the -ing form ("breaking"), and update this scale break explanation. My suggestion is:

`A scale break is an area across an axis that is displayed instead of a section of an axis range. It allows you to improve the readability of a chart section with a large gap in its range.`

Let me know what you think, and feel free to suggest an alternative as well.

Author: Hello, your example seems valid. I would only fix singular: 'readability of chart sections with large gaps in their range'.

Reviewer 2: Thank you. Can we also make "range" plural ("ranges")? For example:

`A scale break is an area across an axis that is displayed instead of a section of an axis range. It allows you to improve the readability of chart sections with large gaps in their ranges.`

![DevExtreme HTML5 JavaScript Charts Scale Breaks](/images/ChartJS/visual_elements/scale-breaks_arg-axis.png)

> Each object in the **breaks** array configures a single scale break. We recommend that a scale break range exceeds the tick interval twice. Otherwise, the scale break might not be visible.

Reviewer: ```suggestion
Each object in the **breaks** array configures a single scale break. We recommend that a scale break range exceeds the tick interval at least twice. Otherwise, the scale break might not be visible.
```

> Each object in the **breaks** array configures a single scale break. We recommend that a scale break range exceeds the tick interval at least twice. Otherwise, the scale break might not be visible.

Reviewer 2: `We recommend that a scale break range exceeds the tick interval at least twice.`

Can you clarify what you mean by "exceeds the tick interval at least twice"?

Author: If a tickInterval is 100, the range between a scale break's startValue and endValue should be at least 200. Like this:
breaks: [{startValue: 300, endValue: 500}]

Reviewer 2: Thanks for the clarification. I think I understand now. What do you think about the following suggestion?

`A scale break range should span at least two tick intervals.`

Author: I'm not sure about the word 'span'. It implies something visual but we are talking about actions to be done in code. So we need to compare two numeric values - a scale break's range and a tick interval.

How about the following?
A scale break should be at least twice as much as a tick interval. Otherwise, ...
